packages/qwik-city/runtime/src/types.ts

```diff
@@ -241,6 +245,25 @@ export interface DocumentLink {
   key?: string;
 }
 
+/**
+ * @alpha
+ */
+export interface DocumentScript {
+  async?: string;
+  crossorigin?: string;
+  defer?: string;
+  fetchpriority?: string;
+  integrity?: string;
+  nomodule?: string;
+  nonce?: string;
+  referrerpolicy?: string;
+  src?: string;
+  type?: string;
+  importmap?: string;
+  blocking?: string;
+  id?: string;
```

Comment:
Should there also be a property to have a script which might go inside the script tag, maybe `script` or `innerHtml` ?

Reply:
I think there is not extra property for that. Please check the official specifications [here](https://html.spec.whatwg.org/multipage/scripting.html#the-script-element)